lib/oracle/oracle.js

```diff
@@ -19,7 +19,24 @@ class Oracle {
   async knownProof(proof) {
     let name = hexEncodeName(proof.name);
     let type = types.toType(proof.type);
-    return (await this.contract.methods.rrdata(type, name).call())[2];
+    let result = await this.contract.methods.rrdata(type, name).call(); 
+    let inception = parseInt(result[0]);
```

Comment:
The result fields are bigints;  you can call `.number()` on them instead of using  parseInt.